crates/red_knot/src/program/mod.rs

```diff
@@ -188,22 +198,30 @@ impl HasJar<LintJar> for Program {
     }
 }
 
+#[derive(Clone, Debug)]
```

Comment:
Seems like a lot to dump all the file changes stuff straight into program/mod.rs. Submodule maybe?

Reply:
Yeah, I don't know how to structure this yet. Maybe I think the file is still reasonably small (Rust has different standards when it comes to large files compared to JS/python ;)).